Replacing YT embeds with Stream embeds (#14153)
cloudflare/cloudflare-docs: content/fundamentals/_partials/_what-is-cloudflare.md

  list: never
---

<div style="position: relative; padding-top: 56.25%;">
    <iframe
        src="https://customer-1mwganm1ma0xgnmj.cloudflarestream.com/bdf5733bed92deded731efee29b2346f/iframe?letterboxColor=transparent&preload=true&poster=https%3A%2F%2Fcustomer-1mwganm1ma0xgnmj.cloudflarestream.com%2Fbdf5733bed92deded731efee29b2346f%2Fthumbnails%2Fthumbnail.jpg%3Ftime%3D70s"
        style="border: none; position: absolute; top: 0; left: 0; height: 100%; width: 100%;"
        allow="accelerometer; gyroscope; autoplay; encrypted-media; picture-in-picture;"
        allowfullscreen="true">
    </iframe>
</div>

> Cloudflare is one of the world’s largest [connectivity cloud networks](https://blog.cloudflare.com/welcome-to-connectivity-cloud). Today, businesses, non-profits, bloggers, and anyone with an Internet presence boast faster, more secure websites and apps thanks to Cloudflare.

Reviewer: ```suggestion
Cloudflare is one of the world’s largest [connectivity cloud networks](https://blog.cloudflare.com/welcome-to-connectivity-cloud). Today, anyone with an Internet presence can have faster and more secure websites and applications thanks to Cloudflare. This includes bloggers, businesses, and even non-profits.
```

Reviewer: Just a suggestion to make the sentence more readable.
